src/components/editor/Editor.jsx

```diff
@@ -0,0 +1,135 @@
+                  "profile:bf2:Title",
+                  "profile:bf2:Title:VarTitle",
+                  "profile:bf2:ParallelTitle",
+                  "profile:bflc:TranscribedTitle"
+                ],
+                "useValuesFrom": [],
+                "valueDataType": {
+                  "remark": ""
+                },
+                "defaults": []
+              },
+              "mandatory": "false",
+              "repeatable": "true",
+              "type": "resource",
+              "remark": ""
+            },
+            {
+              "propertyLabel": "Statement of Responsibility Relating to Title Proper (RDA 2.4.2)",
+              "remark": "http://access.rdatoolkit.org/2.4.2.html",
+              "propertyURI": "http://id.loc.gov/ontologies/bibframe/responsibilityStatement",
+              "mandatory": "false",
+              "repeatable": "true",
+              "type": "literal",
+              "resourceTemplates": [],
+              "valueConstraint": {
+                "valueTemplateRefs": [],
+                "useValuesFrom": [],
+                "valueDataType": {},
+                "defaults": []
+              }
+            },
+            {
+              "propertyURI": "http://id.loc.gov/ontologies/bibframe/issuance",
+              "propertyLabel": "Mode of Issuance (RDA 2.13)",
+              "remark": "http://access.rdatoolkit.org/2.13.html",
+              "mandatory": "false",
+              "repeatable": "true",
+              "type": "resource",
+              "resourceTemplates": [],
+              "valueConstraint": {
+                "valueTemplateRefs": [],
+                "useValuesFrom": [
+                  "http://id.loc.gov/vocabulary/issuance"
+                ],
+                "valueDataType": {
+                  "dataTypeURI": "http://id.loc.gov/ontologies/bibframe/Issuance"
+                },
+                "editable": "false",
+                "repeatable": "true",
+                "defaults": [
+                  {
+                    "defaultURI": "http://id.loc.gov/vocabulary/issuance/mono",
+                    "defaultLiteral": "single unit"
+                  }
+                ]
+              }
+            },
+            {
+              "propertyLabel": "Notes about the Instance",
+              "remark": "http://access.rdatoolkit.org/2.17.html",
+              "propertyURI": "http://id.loc.gov/ontologies/bibframe/note",
+              "mandatory": "false",
+              "repeatable": "true",
+              "type": "resource",
+              "resourceTemplates": [],
+              "valueConstraint": {
+                "valueTemplateRefs": [
+                  "profile:bf2:Note"
+                ],
+                "useValuesFrom": [],
+                "valueDataType": {},
+                "defaults": []
+              }
+            }
+          ]
+        }
+      ]
+    }
+  }
+
+  render() {
+    return(
+      <div id="editor">
+        <h2>Editor Page</h2>
```

Comment:
This `<h2>...` could be defined as =`EditorTitle` component and would make it easier to change the title when different profiles are loaded.